### PR TITLE
Egor_Barilo_Homework1_Task1

### PR DESCRIPTION
**Pull request by barilo-egor**
objects

### CODE REVIEW

**Review by maximkliashchou — APPROVED** [review]
lgfm

**Review by Alexander-1313 — APPROVED** [review]
lgfm

**Review by TLittleProgrammer — APPROVED** [review]
lgfm

File: students/Egor_Barilo/homework1/task1/objects [hunk 1]
```diff
@@ -0,0 +1,90 @@
+Объект - Компьютерная мышь
+  Состояние:
+    Проводная, беспроводная
+    Количество кнопок
+    Тип датчика
+  Поведение:
+    Подключаться к ПК
+    Перемещенать курсора
```
Comment (javaGuruBY) [inline students/Egor_Barilo/homework1/task1/objects:8]: ))

**Review by javaGuruBY — APPROVED** [review]

**Pull request merged**